packages/loader/container-loader/src/loader.ts

```diff
@@ -289,17 +303,25 @@ export class Loader implements IHostLoader {
             loaderId: uuid(),
             loaderVersion: pkgVersion,
         };
+
+        const subMc = mixinMonitoringContext(
+            DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all: telemetryProps }),
```

Comment:
Tangent:  I wish I could magically change this to just `"fluid"` namespace, the `":telemetry"` is just unnecessary